rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_storage.hpp

```diff
@@ -92,6 +92,7 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   void fill_topics_and_types();
   void activate_transaction();
   void commit_transaction();
+  void write_locked(std::shared_ptr<const rosbag2_storage::SerializedBagMessage> message);
```

Comment:
Where is this function used? Since it's private I don't see how it can connect to any existing code, so I'm confused